src/Pages/Login/Login.jsx

```diff
@@ -0,0 +1,44 @@
+import { Navbar } from "../../Components/Navbar/Navbar";
+import { Link } from "react-router-dom";
+import "./login.css";
+const Login = () => {
+  return (
+    <div>
+      <Navbar />
```

Comment:
Suggestion: See if you can take advantage of React Router outlet, so that common components need not be included everywhere